pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/NamespacesBase.java

```diff
@@ -499,7 +488,7 @@ protected void internalDeleteNamespaceForcefully(AsyncResponse asyncResponse, bo
             return;
         }
 
-        FutureUtil.waitForAll(bundleFutures).handle((result, exception) -> {
+        FutureUtil.waitForAll(bundleFutures).thenCompose(__ -> internalClearZkSources()).handle((result, exception) -> {
```

Comment:
why use ``handle`` here?  looks like we don't need return value.

Reply:
This pr only fix the order of `delete system topic and bundle `. And make `internalClearZkSources` async.
The `handle` logic use the before logic.